### DISCUSSION TITLE
Which base to use?

### DISCUSSION THREAD

**Discussion by rchl** [hassio-addons/addon-base#85] [General]
I'm just confused a bit because I see that none of the official addons at https://github.com/home-assistant/addons use this base image. I can see those being used there:

 - homeassistant/[arch]-base
 - homeassistant/[arch]-base-debian
 - homeassistant/[arch]-homeassistant-base
 - ...

So what is the recommended one to use for user-created addons? Those listed above or the one from this repo?

Also, what are the actual tags of images from this repo? I don't see those being listed anywhere or any links to docker hub.

So I guess the point of this issue is to clarify some things to me and ideally put that information in the readme.

**Comment by frenck** [comment]
The core add-ons do use base images?

<https://github.com/home-assistant/addons/blob/master/zwave_js/build.json>

Different ones though. This project uses its own base images.

All images provided by this project (including the base images) are hosted on the GitHub Container Registry (Not DockerHub), click on the "Packages" menu item: <https://github.com/orgs/hassio-addons/packages?type=source>

More specific, filtered by base images: <https://github.com/orgs/hassio-addons/packages?type=source&q=base>


Additionally, GitHub links the container packages with the repository:

In the sidebar of this base repository:

![image](https://user-images.githubusercontent.com/195327/107749307-a6cd9400-6d1a-11eb-8d2d-3fde27314e51.png)


So the ultimate question: Which base image to use? That depends on needs and personal preference 🤷

**Reply by rchl** [reply]
I see. That makes sense. I'm so used to Dockerhub for everything Docker that I didn't even consider those being hosted on Github.

Just a little note in the readme linking to https://github.com/orgs/hassio-addons/packages?repo_name=addon-base could maybe save some others from being confused.

BTW. To add to the confusion, there is also this base image on Docker-hub that I saw being used in some addons: `hassioaddons/base`

**Reply by frenck** [reply]
Yeah, that is an old reference, that pre-dates the GitHub Container Registry. The flow of issues from DockerHub's API limits made it impossible for me to stay there (users got rate-limited and had no clue what to do, tests build failing/limited development).

I'll make sure I'll update the readme to include a list of images 👍
